docusaurus.config.js

```diff
@@ -48,8 +48,11 @@ const config = {
           lastVersion: 'current',
           versions: {
             current: {
+              label: 'v0.5.x',
```

Comment:
I thought we would wait with publishing until 0.5.0 was released? Also, my idea was to only rename the current version's docs, and keep the previous versions as is. I think if we make it clear that there was a rename (e.g. by adding a banner to older docs), that may be the least confusing. Thoughts?

Reply:
@TimoGlastra I have marked 0.5 as unreleased and set the default version to 0.4. This means that 'current' is still 0.5, but users visiting for the first time (clean local storage) will be presented with 0.4.

In order to set 0.4 as the current version we would have to put the 0.5 in the `versioned_docs` directory (next to 0.3) and have 0.4 live in the 'main' directories, which would become a bit messy when 0.5 is released.

We may want to consider moving everything into `versioned_docs` at some point for clarity. Right now we have multiple directories in the root that belong to the current version. We could refactor this so we have a `versioned_docs` directory that looks like this:
- `current/`
- `0.4/`
- `0.3/`

Reply:
@TimoGlastra one more thing regarding the versions. I've only applied the renames to the `0.5` versions of the docs. Do you want me to apply those to `0.4` as well? Since we will release `0.5` pretty soon, I figured adding a banner informing visitors about the rename instead. I want to avoid confusing people with documentation texts that says Credo and install commands that say AFJ.

Resolving for now. Feel free to unresolve if you don't agree.